react/features/base/participants/components/styles.js

```diff
@@ -1,9 +1,32 @@
-import { createStyleSheet } from '../../styles';
+import { BoxModel, ColorPalette, createStyleSheet } from '../../styles';
 
 /**
  * The styles of the feature base/participants.
  */
 export default createStyleSheet({
+    /**
+     * Style for the text rendered when there is a connectivity problem.
+     */
+    interruptedText: {
+        color: ColorPalette.white,
+        fontSize: 12,
+        marginVertical: BoxModel.margin,
+        marginHorizontal: BoxModel.margin / 2,
+        textAlign: 'center'
+    },
+
+    /**
+     * Style for the container of the text rendered when there is a
+     * connectivity problem.
+     */
+    interruptedTextContainer: {
+        alignSelf: 'center',
+        backgroundColor: ColorPalette.darkGrey,
+        borderRadius: 20,
+        marginTop: BoxModel.margin,
+        width: 300
```

Comment:
is it possible to not use constant for 'width' ? wouldn't it make sense to have higher width on horizontal view ?

Reply:
I'll try something like maxWidth 80% or something alike. The large avatar is 200, so having the width as 300 makes it look pretty good.

Reply:
I tried several things but could never get it to look nice, so I'm sticking to 300 (plus a comment) unless you have some better ideas.

Using a percentage wasn't good because in landscape the view would go behind the filmstrip, and looked pretty bad...

Reply:
hm I really don't really like that, but will consult with @lyubomir

Reply:
I think this could be solved by having a container which holds both avatar and this text. Then add left and right margins to the avatar, so that it will stretch the container a little bit. This way the layout would be flexible ?

Reply:
Maybe, but the avatar size is already a constant, so I'm not sure how much we'd gain... anyway, flexbox is not my forte, so I'm definitely open to improvements!

Reply:
So maybe make it at least "avatar size" + 2x box margin or something like that ?

Reply:
Here is what number comes from: https://github.com/jitsi/jitsi-meet/blob/master/react/features/large-video/components/styles.js#L13

I can't use it as-is. I've begun to think we might need to split ParticipantView into 2, one for large view and one for thumbnail. Or take that as a prop or something.

I can add a FIXME, I think this separation will become clearer when we add the display name to the thumbnail. WDYT?